remove_trello_data.py

```diff
@@ -1,93 +1,119 @@
-import sys, getopt
+import configparser
+import getopt
+import sys
 
-from trello_utils import *
 from trello_api import archiveCard, deleteCard, getCards
+from trello_utils import (
+    filterByState,
+    filterByTime,
+    getFollowUpList,
+    getNeedsMetList,
+    getPrepareArchiveDate,
+    getPrepareDeleteDate,
+)
```

Comment:
From [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports):

> Wildcard imports (`from <module> import *`) should be avoided, as they make it unclear which names are present in the namespace, confusing both readers and many automated tools.

Another option (and my personal preference) would be to import the module, and use it as a namespace:

```python
import trello_utils

def prepareForArchive(card, daysSince):
    return trello_utils.filterByTime(card, ...)
```